python/llm/example/GPU/HuggingFace/LLM/chatglm3/README.md

```diff
@@ -146,3 +150,4 @@ Arguments info:
 - `--repo-id-or-model-path REPO_ID_OR_MODEL_PATH`: argument defining the huggingface repo id for the ChatGLM3 model to be downloaded, or the path to the huggingface checkpoint folder. It is default to be `'THUDM/chatglm3-6b'`.
 - `--question QUESTION`: argument defining the question to ask. It is default to be `"晚上睡不着应该怎么办"`.
 - `--disable-stream`: argument defining whether to stream chat. If include `--disable-stream` when running the script, the stream chat is disabled and `chat()` API is used.
+- `--modelscope`: using **Modelscope** as model hub instead of **huggingface**.
```

Comment:
Modelscope -> ModelScope
huggingface -> Hugging Face

Also add description here to remind users of using related repo id if with `--modelscope` option